VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -8,6 +8,9 @@ namespace VulnerableApplication.Controllers
 {
     public class HomeController : Controller
     {
+        public static string SonarPassword = "foo";
+        public static string GitHubPassword = "bar";
```

Comment:
[VULNERABILITY] Make this field 'private' and encapsulate it in a 'public' property.